.github/workflows/cli.yaml

```diff
@@ -17,7 +17,12 @@ permissions:
     packages: write
 jobs:
     build:
+        name: release deb
         runs-on: ubuntu-latest
+        env:
+          MYCLI_DEB_KEY: ${{ secrets.GH_MYCLI_DEB_KEY }}
```

Comment:
rename to `SUNODO_DEB_KEY` and `secrets.SUNODO_DEB_KEY`

Reply:
Fixed !